src/filterlist.cpp

```diff
@@ -424,7 +425,7 @@ void FilterList::checkCriteria()
 
 void FilterList::editCategories()
 {
-  CategoriesDialog dialog(qApp->activeWindow());
+  CategoriesDialog dialog(&m_core.pluginContainer(), qApp->activeWindow());
```

Comment:
See previous comment, I feel like we should not need to pass the plugin container.